api/v1/users/UserHandler.inc.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+/**
+ * @file api/v1/user/UserHandler.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class UserHandler
+ * @ingroup api_v1_user
+ *
+ * @brief Handle API requests for user operations.
+ *
+ */
+
+import('lib.pkp.classes.handler.APIHandler');
+import('classes.core.ServicesContainer');
+
+class UserHandler extends APIHandler {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->_handlerPath = 'users';
+		$roles = array(ROLE_ID_SITE_ADMIN, ROLE_ID_MANAGER);
```

Comment:
Description for `Users` in `swagger.yml` says `Only admins, journal managers and editors can access this endpoint`. I am not very familiar with all the`roles` in OJS but I wonder if you had forgotten to add a last role (may be `ROLE_ID_SUB_EDITOR` ?).